Resources/doc/index.rst

```diff
@@ -125,6 +125,7 @@ The following annotations are defined by the bundle:
 
 This example shows all the available annotations in action::
 
+    use Symfony\Bundle\FrameworkBundle\Controller\Controller;
```

Comment:
Or just add `// ...` to indicate that the example isn't complete (like we do it in the Symfony docs). I mean the namespace declaration is missing too.